classes/FeatureValue.php

```diff
@@ -184,14 +184,18 @@ public static function addFeatureValueImport($idFeature, $value, $idProduct = nu
             return (int) $idFeatureValue;
         }
 
-        // Feature doesn't exist, create it
-        $feature_value = new FeatureValue();
-        $feature_value->id_feature = (int) $idFeature;
-        $feature_value->custom = (bool) $custom;
-        $feature_value->value = array_fill_keys(Language::getIDs(false), $value);
-        $feature_value->add();
+        if($add_if_not_exists) {
```

Comment:
```suggestion
        if ($add_if_not_exists) {
```

The PHP Cs Fixer will be red, as this.

Reply:
My bad

Reply:
There is somes others part as this inside the changes ; doesn't hesitate to make a whole "fix" (https://devdocs.prestashop-project.org/8/modules/testing/basic-checks/#syntax-check)